tools/xml_diff.c

```diff
@@ -158,6 +158,13 @@ main(int argc, char **argv)
         crm_help('?', EX_USAGE);
     }
 
+    if (apply && no_version) {
+        fprintf(stderr, "warning: -u/--no-version ignored with -p/--patch\n");
+    } else if (as_cib && no_version) {
+        fprintf(stderr, "error: -u/--no-version incompatible with -c/--cib\n");
```

Comment:
AFAICS,  --cib in this case  basically determines to  add a digest in the diff. It doesn't have to be incompatible with --no-version.